src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final double defaultSize = 10;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+    private final int randomSize = random.nextInt(10);
+
+    public int getRandomSize() {
+        return randomSize;

```

Comment:
private method should be placed after all public methods
make 10 as a constant

```suggestion
    private int getRandomSize() {
        return random.nextInt(10);
```